third_party/proto/tendermint/abci/types.proto

```diff
@@ -222,7 +222,7 @@ message ResponseDeliverTx {
   int64          gas_wanted = 5 [json_name = "gas_wanted"];
   int64          gas_used   = 6 [json_name = "gas_used"];
   repeated Event events     = 7
-      [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"];
+      [(gogoproto.nullable) = false, (gogoproto.jsontag) = "events,omitempty"]; // nondeterministic
```

Comment:
Result of a proto-gen update

Reply:
what is non-deterministic?

Reply:
Different nodes in the network might have a different view of this data. Ctrl-F in this [doc](https://docs.tendermint.com/master/spec/abci/abci.html). Seems like they just added an explicit comment.